app/shared/footer/FooterContent.js

```diff
@@ -56,6 +60,9 @@ function FooterContent({ t }) {
                 {feedbackLink}
               </p>
             </Col>
+            <Col className="app-varaamo-version-wrapper" lg={3} md={3}>
```

Comment:
You should never put custom classNames and styles for Row and Column components. Rather you can add a div or other wrapping element inside Column.